client/src/components/Ticketing/GroupeToggle.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+
+const GroupToggle = () => {
+  const [filterAs, setFilter] = React.useState('Active');
+
+  const handleFilterChange = (
+    event: React.MouseEvent<HTMLElement>,
+    newFilterSetting: string,
+  ) => {
+    setFilter(newFilterSetting);
+  };
+
+  return (
+    <ToggleButtonGroup
+      color='standard'
+      value={filterAs}
+      exclusive
+      onChange={handleFilterChange}
+      aria-label='filterSetting'
```

Comment:
`aria-label`s are essentially supposed to be read outloud by a screen reader so it should be something understandable yet simple. I'd probably recommend something like 'activeness toggle filter'